docs/.gitignore

```diff
@@ -5,3 +5,5 @@ _sidebar.yml
 /.quarto/
 objects.json
 site_libs/
+_objects_core.json
+_objects_express.json
```

Comment:
@schloerke is it useful to keep these around? Or should we remove them as a part of `python _combine_objects_json.py`?